cli/f3.go

```diff
@@ -0,0 +1,168 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/go-f3/manifest"
+)
+
+var (
+	F3Cmd = &cli.Command{
+		Name:  "f3",
+		Usage: "Manages Filecoin Fast Finality (F3) interactions",
+		Subcommands: []*cli.Command{
+			{
+				Name:    "list-participants",
+				Aliases: []string{"lp"},
+				Usage:   "Lists the miners that currently participate in F3 via this node.",
+				Action: func(cctx *cli.Context) error {
+					api, closer, err := GetFullNodeAPIV1(cctx)
+					if err != nil {
+						return err
+					}
+					defer closer()
+
+					participants, err := api.F3ListParticipants(cctx.Context)
+					if err != nil {
+						return cli.Exit(fmt.Errorf("listing participants: %w", err), 1)
+					}
+					if len(participants) == 0 {
+						_, _ = fmt.Fprintln(cctx.App.Writer, "No participants.")
+						return nil
+					}
+					for i, participant := range participants {
+						_, _ = fmt.Fprintf(cctx.App.Writer, "Participant %d:\n", i+1)
+						_, _ = fmt.Fprintf(cctx.App.Writer, "  Miner ID:         %d\n", participant.MinerID)
+						_, _ = fmt.Fprintf(cctx.App.Writer, "  From Instance:    %d\n", participant.FromInstance)
+						_, _ = fmt.Fprintf(cctx.App.Writer, "  Validity Term:    %d\n", participant.ValidityTerm)
+						_, _ = fmt.Fprintf(cctx.App.Writer, "\n")
```

Comment:
I'd print this as a table. See https://github.com/filecoin-project/lotus/blob/fda61d33b30973512afb26aff2dce007db4ec313/cli/wallet.go#L78.

I'd also print it as: ID, From, To (easier to compare across the different participants).